modules/custom/az_demo/data/az_demo_splitscreen_paragraph.json

```diff
@@ -136,6 +136,69 @@
+            "filename" : "study.jpeg",
+            "full_width" : false,
+            "ordering" : "order_0",
+            "bg_color" : "bg-blue",
+            "bottom_spacing" : "mb-0"
+        },
+        {
+            "id":73,
+            "az_text_area" : "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<\/p>",
+            "filename" : "arizona_campus_fireworks.jpg",
+            "full_width" : "full-width-background",
+            "ordering" : "order_1",
+            "bg_color" : "bg-chili",
+            "bottom_spacing" : "mb-0"
+        },
+        {
+            "id":74,
+            "az_text_area" : "<h2>Gray 100 Background<\/h2><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<\/p><p><a class=\"btn btn-blue\" href=\"#\">Button<\/a><\/p>",
+            "filename" : "lynx.jpg",
+            "full_width" : "full-width-background",
+            "ordering" : "order_1",
+            "bg_color" : "bg-gray-100",
+            "bottom_spacing" : "mb-0"
+        },
+        {
+            "id":75,
+            "az_text_area" : "<h2>Gray 300 Background<\/h2><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.<\/p><p><a class=\"btn btn-blue\" href=\"#\">Button<\/a><\/p>",
+            "filename" : "june_bugs.jpeg",
+            "full_width" : false,
+            "ordering" : "order_0",
+            "bg_color" : "bg-gray-300",
+            "bottom_spacing" : "mb-0"
```

Comment:
```suggestion
```

Reply:
I don't think they are necessary since the other gray backgrounds will be on the text with background page

Reply:
You'll want to remove the reference to those ids in the page node migration as well.